tasks/configure-Debian.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Create /etc/default/kibana
+  template:
+    src: Ubuntu.default.j2
+    dest: /etc/default/kibana
+    # any files under /etc/default should be mode 644, but the package installs
+    # the file as mode 0664
```

Comment:
i simply do not know why the mode is 664. not an expert in Debian/Ubuntu in any way, but here is my understanding: files under `/etc/default` are supposed to be sourced by init scripts, or systemd unit files. in rare cases, such as when the content must be protected from non-previledged users, different mode should be used. but 664? no, it does not make sense in this context. if a member of `kibana` needs write access to files under `/etc/default`, that should be a bug.

no, i do not know if the issue has been reported as a bug.